src/timer/CountDownTimer.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import { Icon, useToggle } from '@edx/paragon';
+import { Visibility, VisibilityOff } from '@edx/paragon/icons';
+
+const TICK = 1;
+const LOW_TIME = 60;
```

Comment:
there should be a value for this on the attempt model returned by the backend. It actually looks like we have two values/style changes for this, low and critically low.